### ISSUE TITLE
Implement UIViewControllerTransitioningDelegate methods in MDMTransitionController

### ISSUE THREAD

**Issue by jverkoey** [material-motion/material-motion-transitions-objc#10]
```
- (id<UIViewControllerAnimatedTransitioning>)animationControllerForPresentedController:(UIViewController *)presented
                                                                  presentingController:(UIViewController *)presenting
                                                                      sourceController:(UIViewController *)source;
```

Must return an instance that conforms to `UIViewControllerAnimatedTransitioning`. This instance should be a separate bridging object that creates the scheduler and director instances and coordinates their overall execution.

```
- (id<UIViewControllerAnimatedTransitioning>)animationControllerForDismissedController:(UIViewController *)dismissed;
```

Similar to the above, but requires a bit of shenanigans to infer the `sourceViewController`.

```
- (id<UIViewControllerInteractiveTransitioning>)interactionControllerForPresentation:(id<UIViewControllerAnimatedTransitioning>)animator;
```

If we're driving this transition, the interaction controller is always the animator instance we returned in the first method.

```
- (id<UIViewControllerInteractiveTransitioning>)interactionControllerForDismissal:(id<UIViewControllerAnimatedTransitioning>)animator;
```

If we're driving this transition, the interaction controller is always the animator instance we returned in the first method.

**Comment by jverkoey** [comment]
Blocked by #11.

**Comment by jverkoey** [comment]
Blocked by #3.

**Comment by jverkoey** [comment]
Landed in 0aa710960343c332ccee59f3284b24dc6dee69f6.
